<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageViewForm</class>
 <widget class="QWidget" name="imageViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_image">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lab_num">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="btn_next">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn_pre">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn_close2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
